protos/meta_service.proto

```diff
@@ -166,20 +166,37 @@ message ShardInfo {
     PartialOpen = 1;
   }
 
+  message Load {
+    uint64 written_bytes = 1;
+    uint64 read_bytes = 2;
+  }
```

Comment:
Define it outside the `ShardInfo`? Otherwise, it may bring trouble when access it.